docs/deliver-to-pypi.md

```diff
@@ -4,33 +4,93 @@ title: Delivering to PyPI
 
 After you've built your wheels, you'll probably want to deliver them to PyPI.
 
-### Manual method
+## Manual method
 
-On your development machine, do the following...
+On your development machine, install [pipx](https://pypa.github.io/pipx/) and do the following:
 
 ```bash
+# Either download the SDist from your CI, or make it:
 # Clear out your 'dist' folder.
 rm -rf dist
 # Make a source distribution
-python setup.py sdist
+pipx run build --sdist
 
 # 🏃🏻
 # Go and download your wheel files from wherever you put them. e.g. your CI
 # provider can be configured to store them for you. Put them all into the
 # 'dist' folder.
 
 # Upload using 'twine' (you may need to 'pip install twine')
-twine upload dist/*
+pipx run twine upload dist/*
 ```
 
-### Semi-automatic method using wheelhouse-uploader
+## Automatic method
 
-Obviously, manual steps are for chumps, so we can automate this a little by using [wheelhouse-uploader](https://github.com/ogrisel/wheelhouse-uploader).
+If you don't need much control over the release of a package, you can set up
+cibuildwheel to deliver the wheels straight to PyPI. You just need to bump the
+version and tag it.
 
-> Quick note from me - using S3 as a storage didn't work due to a [bug](https://issues.apache.org/jira/browse/LIBCLOUD-792) in libcloud. Feel free to use my fork of that package that fixes the bug `pip install https://github.com/joerick/libcloud/archive/v1.5.0-s3fix.zip`
+### Generic instructions
 
-### Automatic method
+You'll want to build your SDist with [build](https://github.com/pypa/build) and your wheels with cibuildwheel. If you can make the files available as
+downloadable artifacts, this make testing before releases easier (depending on your CI provider's options). The "publish" job/step should collect the
+files, and then run `twine upload <paths>` (possibly via [pipx](https://github.com/pypa/pipx)); this should only happen on tags or "releases".
```

Comment:
The 'build' tool name makes this sentence quite hard to read... perhaps this is better.

```suggestion
Make your SDist with the [build](https://github.com/pypa/build) tool, and your wheels with cibuildwheel. If you can make the files available as
downloadable artifacts, this make testing before releases easier (depending on your CI provider's options). The "publish" job/step should collect the
files, and then run `twine upload <paths>` (possibly via [pipx](https://github.com/pypa/pipx)); this should only happen on tags or "releases".
```